src/node_perf.cc

```diff
@@ -524,7 +524,7 @@ static void ELDHistogramReset(const FunctionCallbackInfo<Value>& args) {
 static void ELDHistogramNew(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
   CHECK(args.IsConstructCall());
-  int32_t resolution = args[0]->IntegerValue(env->context()).FromJust();
```

Comment:
#37155 changes the implementation of `ELDHistogram` already anyway, eliminating this function. I'd just remove this change from this PR.